d3-radar/src/App.js

```diff
@@ -0,0 +1,220 @@
+	// 	source: 'group input 2',
+	// 	type: 'text'
+	// },
+	// {
+	// 	name: 'text input 4',
+	// 	source: 'group input 2',
+	// 	type: 'text'
+	// },
+	// {
+	// 	name: 'secure input 5',
+	// 	type: 'text',
+	// 	secure: true
+	// },
+	// {
+	// 	name: 'checkbox input 6',
+	// 	type: 'checkbox'
+	// },
+	// {
+	// 	name: 'toggle input 7',
+	// 	type: 'toggle'
+	// },
+]);
+
+function chunk(data, colInfo, sets, variables, numGroups) {
+	if (sets && variables && data && colInfo) {
+		// let valuesObj = {};
+		// for(let i = 0; i < variables.length; i++) {
+		// 	valuesObj[variables[i]] = Math.random() * 10;
+		// }
+
+		const setCol = data[sets] ?? [];
+		const uniqueSets = setCol.filter(onlyUnique).slice(0, numGroups);
+		const setList = uniqueSets.map((name) => {
+			let valuesObj = {};
+			for(let i = 0; i < variables.length; i++) {
+				valuesObj[variables[i]] = 0;
+			}
+			return (
+				{
+					key: name,
+					label: name,
+					values: {
+						...valuesObj
+					},
+				}
+			);
+		})
+		// console.log(variables); // list of spoke variable col names
+		const variableObjList = variables.map((variableName) => {
+			return {
+				key: variableName,
+				label: colInfo[variableName]?.name,
+			}
+		})
+
+		for(let j = 0; j < setCol.length; j++){
+			let thisSet = setCol[j];
+			if (uniqueSets.includes(thisSet)) {
+				for(let setIdx = 0; setIdx < uniqueSets.length; setIdx++){
+					if(setList[setIdx].key === thisSet) {
+						for(let varIdx = 0; varIdx < variables.length; varIdx++){
+							setList[setIdx].values[variables[varIdx]] += data[variables[varIdx]][j];
+						}
+					}
+				}
+			}
+		}
+
+		return {
+			variables: variableObjList,
+			sets: setList,
+		}
+	} else {
+		return (
+			{
+				variables: [
+					{ key: 'resilience', label: 'Resilience' },
+					{ key: 'strength', label: 'Strength' },
+					{ key: 'adaptability', label: 'Adaptability' },
+					{ key: 'creativity', label: 'Creativity' },
+					{ key: 'openness', label: 'Open to Change' },
+					{ key: 'confidence', label: 'Confidence' },
+				],
+				sets: [
+					{
+						key: 'me',
+						label: 'My Scores',
+						values: {
+							resilience: 4,
+							strength: 6,
+							adaptability: 7,
+							creativity: 2,
+							openness: 8,
+							confidence: 1,
+						},
+					},
+					{
+						key: 'everyone',
+						label: 'Everyone',
+						values: {
+							resilience: 10,
+							strength: 8,
+							adaptability: 6,
+							creativity: 4,
+							openness: 2,
+							confidence: 3,
+						},
+					},
+					{
+						key: 'everyone2',
+						label: 'Everyone2',
+						values: {
+							resilience: 1,
+							strength: 6,
+							adaptability: 3,
+							creativity: 1,
+							openness: 6,
+							confidence: 9,
+						},
+					},
+				],
+			}
+		);
+	}
+}
+
+function onlyUnique(value, index, array) {
+	return array.indexOf(value) === index;
+  }
+
+function App() {
+	// useConfig, useElementColumns, useElementData is a function of the Sigma API
```

Comment:
remove. you are importing these functions from `@sigmacomputing/plugin` so its already clear that these are part of the sigma API